commands.json

```diff
@@ -57,7 +57,7 @@
       }
     ],
     "since": "2.6.0",
-    "group": "string"
+    "group": "bitmap"
```

Comment:
I suggest changing to "bitstring" because the word bitmap often refers to images. But I guess that's not even visible externally.

Reply:
They're called bit arrays or bitmaps everywhere else in our documentation, https://redis.io/topics/data-types-intro, seems like we should stick with bitmaps.